app/models/employee.rb

```diff
@@ -2,4 +2,12 @@ class Employee < ActiveRecord::Base
   validates :slack_username, presence: true, uniqueness: true, format: { with: /\A[a-z_0-9.]+\z/, message: "Slack usernames can only contain lowercase letters, numbers, underscores, and periods." }
   validates :started_on, presence: true
   validates :time_zone, presence: true
+
+  def self.filter(params)
+    results = self.all.where("slack_username like ?", "%#{params[:slack_username]}%")
+    if params[:started_on].present?
+      results.where(started_on: params[:started_on])
+    end
+    return results
```

Comment:
@jessieay:  Do you mean the explicit `return` statement or the whole method?  If just the `return` line, will `results` be what's returned, including if the `if` statement evaluates to true?  I know Ruby will automatically return whatever the result was of the last statement in a method, but sometimes it's just not entirely clear from a readability perspective.